src/main/java/uk/gov/hmcts/reform/preapi/services/AuditService.java

```diff
@@ -43,6 +46,13 @@ public UpsertResult upsert(CreateAuditDTO createAuditDTO, @Nullable UUID created
         return UpsertResult.CREATED;
     }
 
+    @Transactional
+    public Page<CreateAuditDTO> findAll(Pageable pageable) {
+        return auditRepository
+            .searchAll(pageable)
+            .map(CreateAuditDTO::new);
```

Comment:
Can you create an AuditDTO instead of using the CreateAuditDTO. This dto needs to have everything that CreateAuditDTO has as well as created at and created by